From 767b0e7534e7d0af6090c105f0d093461d5ea03a Mon Sep 17 00:00:00 2001
From: Miroslav Rezanina <mrezanin@redhat.com>
Date: Wed, 18 Sep 2013 10:39:19 +0200
Subject: [PATCH 26/29] virtio-scsi: Make type virtio-scsi-common abstract

RH-Author: Miroslav Rezanina <mrezanin@redhat.com>
Message-id: <b7aa915feada79f108cf99b4a67c7168b70c5593.1379500275.git.mrezanin@redhat.com>
Patchwork-id: 54440
O-Subject: [RHEL7 qemu-kvm PATCH] virtio-scsi: Make type virtio-scsi-common abstract
Bugzilla: 903918
RH-Acked-by: Markus Armbruster <armbru@redhat.com>
RH-Acked-by: Stefan Hajnoczi <stefanha@redhat.com>
RH-Acked-by: Paolo Bonzini <pbonzini@redhat.com>

From: Markus Armbruster <armbru@redhat.com>

BZ: 903918
Brewbuild: https://brewweb.devel.redhat.com/taskinfo?taskID=6298357

It's the abstract base of virtio-scsi-device and vhost-scsi.

Signed-off-by: Markus Armbruster <armbru@redhat.com>
Signed-off-by: Paolo Bonzini <pbonzini@redhat.com>
(cherry picked from commit a27292b5d7545509bfa171922516d2033c570205)

Clean backport
Signed-off-by: Miroslav Rezanina <mrezanin@redhat.com>
---
 hw/scsi/virtio-scsi.c | 1 +
 1 file changed, 1 insertion(+)

Signed-off-by: Miroslav Rezanina <mrezanin@redhat.com>
---
 hw/scsi/virtio-scsi.c |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/hw/scsi/virtio-scsi.c b/hw/scsi/virtio-scsi.c
index 08dd3f3..4074088 100644
--- a/hw/scsi/virtio-scsi.c
+++ b/hw/scsi/virtio-scsi.c
@@ -682,6 +682,7 @@ static const TypeInfo virtio_scsi_common_info = {
     .name = TYPE_VIRTIO_SCSI_COMMON,
     .parent = TYPE_VIRTIO_DEVICE,
     .instance_size = sizeof(VirtIOSCSICommon),
+    .abstract = true,
     .class_init = virtio_scsi_common_class_init,
 };
 
-- 
1.7.1

